### PR TITLE
Add example kustomization which gives cAdvisor access to NVML

### PR DESCRIPTION
**Pull request by dashpole**
I have gotten a bunch of questions about how to use containerized cAdvisor to monitor GPUs.  This should give _something_ for me to point at, even if it isn't perfect.

cc @mindprince 
/assign @tallclair

### CODE REVIEW

File: deploy/kubernetes/overlays/examples/gpu-privilages.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# This patch adds the required hostpath mounts and privilages to initialize NVML, as well as to access devices.
+apiVersion: apps/v1 # for Kubernetes versions before 1.9.0 use apps/v1beta2
```
Comment (mindprince) [inline deploy/kubernetes/overlays/examples/gpu-privilages.yaml]: I would skip the comment here.

File: deploy/kubernetes/overlays/examples/gpu-privilages.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# This patch adds the required hostpath mounts and privilages to initialize NVML, as well as to access devices.
```
Comment (mindprince) [inline deploy/kubernetes/overlays/examples/gpu-privilages.yaml]: Maybe link to https://github.com/google/cadvisor/blob/master/docs/running.md#hardware-accelerator-monitoring from here?

File: deploy/kubernetes/overlays/examples/gpu-privilages.yaml (hunk 1 above)
Comment (dashpole) [reply deploy/kubernetes/overlays/examples/gpu-privilages.yaml]: done

File: deploy/kubernetes/overlays/examples/gpu-privilages.yaml (hunk 2 above)
Comment (dashpole) [reply deploy/kubernetes/overlays/examples/gpu-privilages.yaml]: done

**Pull request merged**